kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/lookup/UaDisbursementPayeeLookupableHelperServiceImpl.java

```diff
@@ -65,15 +71,47 @@
 	protected List<DisbursementPayee> getPersonAsPayeesByEmployeeStatusCode(Map<String, String> fieldValues, String employeeStatusCode) {
 		List<DisbursementPayee> payeeList = new ArrayList<DisbursementPayee>();
 
-		fieldValues.put(KIMPropertyConstants.Person.EMPLOYEE_STATUS_CODE, employeeStatusCode);
-		List<? extends Person> persons = KimApiServiceLocator.getPersonService().findPeople(fieldValues);
+		Map<String, String> personFieldValues = this.getPersonFieldValues(fieldValues);
+		personFieldValues.put(KIMPropertyConstants.Person.EMPLOYEE_STATUS_CODE, employeeStatusCode);
+		List<? extends Person> persons = getPersonService().findPeople(personFieldValues);
 
 		for (Person personDetail : persons) {
-			DisbursementPayee payee = getPayeeFromPerson(personDetail, fieldValues);
+			DisbursementPayee payee = getPayeeFromPerson(personDetail, personFieldValues);
 			payeeList.add(payee);
 		}
 
 		return payeeList;
 	}
 
+	private static volatile PersonService personService;
+
+	private PersonService getPersonService() {
+		if (personService == null) {
+			personService = SpringContext.getBean(PersonService.class);
+		}
+		return personService;
+	}
+
+	@Override
+	protected Map<String, String> getPersonFieldValues(Map<String, String> fieldValues) {

```

Comment:
The field name conversions aren't needed because the field names being used are the same field names coming out of the EDS. The field name conversion method was essentially doing a "change field name firstName to firstName".